tests/CMakeLists.txt

```diff
@@ -68,5 +68,7 @@ mdspan_add_test(test_mdarray_to_mdspan)
 endif()
 
 if(CMAKE_CXX_STANDARD GREATER_EQUAL 20)
+if((CMAKE_CXX_COMPILER_ID STREQUAL Clang) OR ((CMAKE_CXX_COMPILER_ID STREQUAL GNU) AND (CMAKE_CXX_COMPILER_VERSION GREATER_EQUAL 12.0.0)))
```

Comment:
should it be `GREATER_EQUAL` -> `VERSION_GREATER_EQUAL`?

Reply:
ah probably VERSION_GREATER_EQUAL hm but its weird it did work on my system. I.e. I checked that with GNU 12 the tests were there and with 11 they were not.